play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -267,7 +275,8 @@ V
 The Trail becomes bright against a monochrome background of edged shades of 
 grey. +75% to a tracking check, or maximum 2 turns to pass.
 ----------------------
-Highlight Target:
+Highlight Target: (Perception checks do not include checks to hit target. But by using a perception check for an 
+action to see where the target is and passing the target''s Cover DC+25 the player may recieve a +5% / +10% accuracy bonus.)
```

Comment:
Idk what this perception stuff is talking about. I can't remember who wrote this but.. I can spend an action to find the target? Is this just a normal perception check, except I can get a bonus from it? And is this a separate ability besides everything below?

Reply:
@bleehu / @trowl223  Did this ever get clarified?